astropy/utils/tests/test_data.py

```diff
@@ -46,6 +46,17 @@ def test_download_nocache():
     assert os.path.isfile(fnout)
 
 
+@remote_data(source='astropy')
+def test_download_parallel():
+    from ..data import download_files_in_parallel
+
+    fnout = download_files_in_parallel(
+        ['http://data.astropy.org',
+         'https://astropy.stsci.edu/data/intersphinx/README'],
+        cache=True)
+    assert all([os.path.isfile(f) for f in fnout]), fnout
```

Comment:
ah, I see I've forgotten to remove the "assert failure message" here (the trailing `, fnout`. Can that stay in or should it be removed (I've never seen it used in other tests).

Reply:
If the value of `fnout` is useful for debugging, no harm leaving it in. But the test will never fail again, right, so this is a non-issue??? 😉

Reply:
I was using it when `cache=False` kept failing to see where the files were "supposed" to be.

I guess this test will fail a lot if we use it with `remote_data('astropy')` (timeout ... yay 🎉) but not in the assert 😄